src/parsers/gas.ts

```diff
@@ -145,6 +146,20 @@ export const parseRainbowMeteorologyData = (
   };
 };
 
+export const parseLegacyRainbowMeteorologyData = (rainbowMeterologyData: MeteorologyLegacyResponse): GasPricesAPIData => {
```

Comment:
is this function needed? I don't see it used anywhere

Reply:
removing